samples/stock/demo/orderbook-chart/demo.js

```diff
@@ -0,0 +1,185 @@
+        backgroundColor: '#23232f',
+        marginTop: 70,
+        events: {
+            load() {
+                this.titles = [];
+                setInterval(() => {
+                    updateData(this.series);
+                }, 200);
+            },
+            render() {
+                // Create and align titles
+                if (this.titles.length) {
+                    this.titles.forEach(title => {
+                        title.destroy();
+                    });
+                    this.titles.length = 0;
+                }
+
+                const { plotLeft, plotTop, plotWidth, renderer } = this;
+
+                this.titles.push(
+                    renderer.text(
+                        'Bids', plotLeft, plotTop - 5
+                    )
+                        .attr({ fill: '#ffffff' })
+                        .css({
+                            fontWeight: 700,
+                            fontSize: 18
+                        })
+                        .add()
+                );
+
+                this.titles.push(
+                    renderer.text(
+                        'Asks', plotWidth - 35, plotTop - 5
+                    )
+                        .attr({ fill: '#ffffff' })
+                        .css({
+                            fontWeight: 700,
+                            fontSize: 18
+                        })
+                        .add()
+                );
+
+                this.titles.push(
+                    renderer.text(
+                        'Price', plotWidth / 2 - plotLeft, plotTop - 5
+                    )
+                        .attr({ fill: '#ffffff' })
+                        .css({
+                            fontWeight: 700,
+                            fontSize: 18
+                        })
+                        .add()
+                );
+            }
+        }
+    },
+
+    title: {
+        text: 'Orderbook live chart',
+        style: {
+            color: '#ffffff'
+        }
+    },
+
+    xAxis: [{
+        reversed: true,
+        visible: false
+    }, {
+        opposite: true,
+        visible: false
+    }],
+
+    yAxis: {
+        visible: false,
+        min: -1200000,
+        max: 1200000
+    },
+
+    legend: {
+        enabled: false
+    },
+
+    navigation: {
+        buttonOptions: {
+            theme: {
+                fill: 'none'
+            }
+        }
+    },
+
+    plotOptions: {
+        series: {
+            animation: false,
+            stacking: 'normal',
+            pointPadding: 0,
+            groupPadding: 0,
+            dataLabels: {
+                enabled: true,
+                color: '#ffffff'
+            },
+            borderWidth: 0
+        }
+    },
+
+    series: [{
+        dataLabels: [{
+            x: -999,
+            style: {
+                fontSize: 14,
+                textOutline: 0
+            },
+            formatter() {
+                return Math.abs(this.point.y);
```

Comment:
Because of that `formatter`, we have mismatching formats in labels. And use `format()` ;) And `price` perhaps should also be formatted (currency + thousand separator)



![Zrzut ekranu 2024-03-19 o 13 39 24](https://github.com/highcharts/highcharts/assets/1453926/1c4a01bf-0b0f-4d25-800d-3c6f16889187)